### ISSUE TITLE
Copy to clipboard feature

### ISSUE THREAD

**Issue by John15321** [knnativeflow/zdalnie-na-pwr#29]
Would be cool if when clicking upon on of the icons like Kod grupy Termin ZOOM link etc, the contents of that (so for example zoom link, kod grupy) got copied to the clipboard
![image](https://user-images.githubusercontent.com/56229312/102027714-58e1f180-3da6-11eb-8373-2cc9fdd40c1a.png)

**Comment by waszkowski** [comment]
Good idea, I added to the list of features to be created in the new year.

**Comment by Drabenstein** [comment]
IMO it would be better if the copy feature would work on right click (maybe menu) or some 'copy icon' next to contents as current zoom link click action is really useful.

**Comment by John15321** [comment]
> IMO it would be better if the copy feature would work on right click (maybe menu) or some 'copy icon' next to contents as current zoom link click action is really useful.

Yeah, an icon would be the best choice

**Comment by waszkowski** [comment]
We are currently thinking about moving the application to the website, then the problem would be solved by itself :D

**Comment by Drabenstein** [comment]
@waszkowski Do you mean making the app remote not desktop? It would help with sync between mutiple devices but in terms of security it would be better to migrate (at least mail) to OAuth2 as current way frequently triggers antivirus alerts about unverified connection.

**Comment by waszkowski** [comment]
I don't know what you mean by remote application. We are thinking of a web application.

The migration to Gmail started with OAuth. You can find the reason why we didn't do this at the link https://www.facebook.com/knnativeflow/posts/977167899359172
